src/views/List.jsx

```diff
@@ -4,14 +4,17 @@ export function List({ data }) {
 	return (
 		<>
 			<p>
-				Hello from the <code>/list</code> page!
+				Hello from the <code>/list</code> page! Our first test
 			</p>
 			<ul>
 				{/**
 				 * TODO: write some JavaScript that renders the `data` array
 				 * using the `ListItem` component that's imported at the top
 				 * of this file.
 				 */}
+				{data.map((item, index) => (
```

Comment:
I'm a fan of adding null checks in so that bad values don't break the component

```suggestion
				{data && data.map((item, index) => (
```
You can simluate this by passing a null in from App.jsx

```
<Route path="/list" element={<List data={null} />} />
```